application/src/main/java/org/togetherjava/tjbot/commands/moderation/meme/MemeListener.java

```diff
@@ -0,0 +1,65 @@
+package org.togetherjava.tjbot.commands.moderation.meme;
```

Comment:
I understand why you put it into `moderation`, but it feels a bit off for me. Just put it into `commands.meme` maybe, a level higher

Reply:
It shouldn't be in an command package in my opinion, it's not a command

Reply:
yes, the package hierarchy is bad. there is a ticket open that says that this should be renamed into `feature` and then we have "by topic" grouping instead.

work on that issue actually even already started, but i pushed it on hold until there is less traffic on the project - cause such a huge renaming would cause massive merge conflicts.

Reply:
moved to commands.meme